measure_extinction/extdata.py

```diff
@@ -1384,20 +1386,32 @@ def plot(
             if wavenum:
                 x = 1.0 / x
 
+            if (legend_key is not None) and (legend_key == curtype):
+                clabel = legend_label
+            else:
+                clabel = None
```

Comment:
I am a little confused.
It looks like with this, you only label 1 part of the curve, i.e. the one that matches what is given in "legend_key". E.g. if you run this with (legend_key="IRS", legend_label="IRS"), it will take the color of the IRS part of the curve and label that in the legend as "IRS". But what if you have multiple parts in the curve that you want to label, e.g. also "SpeX"? Or what exactly would you use as the legend_label?

Reply:
The idea here is that I wanted only one label for all the data.  Just to say this is the HD283809 curve.  Not each part.  See the plot I posted in slack.  This could be expanded, but at least now it does *something*.  ;-)

Reply:
I see. There is a better way to add a legend with just 1 label for all the data that does not require the "legend_key". You could just add a legend at the end (outside of the loop) with 1 handle and 1 label. This is just a suggestion. You can leave it for now ;).